gen_config.py

```diff
@@ -0,0 +1,161 @@
+import typing as t
```

Comment:
this will either be added to neuro/apolo-extras, separate package or added to app deployment controller, wdyt?